pilco/controllers.py

```diff
@@ -107,8 +127,8 @@ def compute_action(self, m, s, squash=True):
     def randomize(self):
         print("Randomising controller")
         for m in self.models:
-            mean = 0; sigma = 0.1
-            m.X.assign(mean + sigma*np.random.normal(size=m.X.shape))
-            m.Y.assign(mean + sigma*np.random.normal(size=m.Y.shape))
+            mean = 0; sigma = 1.0
+            m.X.assign(mean + sigma * np.random.normal(size=m.data[0].shape))
+            m.Y.assign(mean + 0.1 * self.max_action * np.random.normal(size=m.data[1].shape))
             mean = 1; sigma = 0.1
-            m.kern.lengthscales.assign(mean + sigma*np.random.normal(size=m.kern.lengthscales.shape))
+            m.kernel.lengthscales.assign(mean + sigma*np.random.normal(size=m.kernel.lengthscales.shape))
```

Comment:
I will also pull `mean` and `sigma` as parameters in `randomise`, unless you suggest otherwise.

Reply:
Sure, I am fine with fine with that.